README.md

```diff
@@ -623,10 +624,23 @@ The most interesting metrics are:
 
 * If VictoriaMetrics doesn't work because of certain parts are corrupted due to disk errors,
   then just remove directoreis with broken parts. This will recover VictoriaMetrics at the cost
-  of data loss stored in the broken parts. In the future `vmrecover` tool will be created
+  of data loss stored in the broken parts. In the future, `vmrecover` tool will be created
   for automatic recovering from such errors.
 
+## Roadmap
 
+The road to 2.0 version:
```

Comment:
version 2.0 means breaking changes. Are we sure about it?

Reply:
I don't think that we can deliver these features without breaking changes, specially replication